### PR TITLE
Small simplification to SelectBinIndex in buffers

### PR DESCRIPTION
**Pull request by VSadov**
Small simplification to SelectBinIndex in buffers

### CODE REVIEW

**Comment by KrzysztofCwalina** [comment]
cc @sokket

File: src/System.Buffers/src/System/Buffers/ArrayBufferPool.cs [hunk 1]
```diff
@@ -87,9 +87,17 @@ public static BufferPool Shared
         private int SelectBinIndex(int bufferSize)
         {
             Precondition.Require(bufferSize > 0);
-            var _poolIndex = MaxBitIndexSet((ulong)bufferSize - 1) - 3;
-            if (_poolIndex < 0) return 0;
-            return _poolIndex;
+
+            // Log2((bufferSize - 1) >> 3)
```
Comment (sokket) [inline src/System.Buffers/src/System/Buffers/ArrayBufferPool.cs]: Is this comment still relevant?

**Comment by sokket** [comment]
Super minor NIT but otherwise LGTM

File: src/System.Buffers/src/System/Buffers/ArrayBufferPool.cs (hunk 1 above)
Comment (VSadov) [inline src/System.Buffers/src/System/Buffers/ArrayBufferPool.cs]: I think it makes it easier to understand what algorithm does.

**Pull request merged**